src/js/messaging/containers/Folder.jsx

```diff
@@ -364,6 +376,7 @@ export class Folder extends React.Component {
       let messageSearch;
       if (this.props.messages && this.props.messages.length || this.props.filter) {
         messageSearch = (<MessageSearch
+            hasRecipientField={folderName === 'Sent' || folderName === 'Drafts'}
```

Comment:
we can use `indexOf` if there are more than 2 elements we need to match in the future